frontend/apps/web/src/pages/behaviors.tsx

```diff
@@ -64,6 +65,74 @@ const Behaviors = () => {
     prepareBehaviorChart(),
   )
   const navigate = useNavigate()
+  const options = {
+    keys: ['description', 'id', 'parentFeatureName'],
+  }
+  const testedOptions = {
+    keys: ['tested'],
+  }
+  const filterOptions = [
+    {
+      label: 'All statuses',
+      value: undefined,
+    },
+    {
+      label: 'Tested',
+      value: 'true',
+    },
+    {
+      label: 'Untested',
+      value: 'false',
+    },
+    {
+      label: 'Unknown',
+      value: 'unknown',
```

Comment:
@aleksaToljic since `tested `property is boolean , i should  place `"undefined"` string here insted of `unknown`?